finesse/hardware/stepper_motor/dummy.py

```diff
@@ -47,8 +47,16 @@ def steps_per_rotation(self) -> int:
         return self._steps_per_rotation
 
     @property
-    def step(self) -> int:
+    def is_moving(self) -> bool:
+        """Whether the motor is currently moving."""
+        return self._move_end_timer.isActive()
+
+    @property
+    def step(self) -> int | None:
```

Comment:
According to the docstring, this one is doing something different than the equivalent in the real device. Is this outdated?

Reply:
Indeed it is. Good spot. I've updated it now.